react/features/invite/reducer.js

```diff
@@ -54,17 +56,24 @@ ReducerRegistry.register('features/invite', (state = DEFAULT_STATE, action) => {
         };
 
     case UPDATE_DIAL_IN_NUMBERS_SUCCESS: {
-        const {
-            defaultCountry,
-            numbers,
-            numbersEnabled
-        } = action.dialInNumbers;
+        if (Array.isArray(action.dialInNumbers)) {
+            return {
+                ...state,
+                conferenceID: action.conferenceID,
+                numbers: action.dialInNumbers,
+                numbersEnabled: true
+            };
+        }
+
+        // this is the old format which is deprecated
+        logger.warn('Using deprecated API for retrieving phone numbers');
```

Comment:
What's the deprecation plan or timeline?

Reply:
None for now.